tables/Tables/test/tTableUtil.cc

```diff
@@ -0,0 +1,276 @@
+//# tTableUtil.cc Test program for table utility functions
+//# Copyright (C) 2022
+//# Associated Universities, Inc. Washington DC, USA.
+//#
+//# This program is free software; you can redistribute it and/or modify it
+//# under the terms of the GNU General Public License as published by the Free
+//# Software Foundation; either version 2 of the License, or (at your option)
+//# any later version.
+//#
+//# This program is distributed in the hope that it will be useful, but WITHOUT
+//# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+//# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+//# more details.
+//#
+//# You should have received a copy of the GNU General Public License along
+//# with this program; if not, write to the Free Software Foundation, Inc.,
+//# 675 Massachusetts Ave, Cambridge, MA 02139, USA.
+//#
+//# Correspondence concerning AIPS++ should be addressed as follows:
+//#        Internet email: aips2-request@nrao.edu.
+//#        Postal address: AIPS++ Project Office
+//#                        National Radio Astronomy Observatory
+//#                        520 Edgemont Road
+//#                        Charlottesville, VA 22903-2475 USA
+//#
+//# $Id$
+
+//# Includes
+#include <casacore/casa/aips.h>
+#include <casacore/tables/Tables/TableUtil.h>
+#include <casacore/tables/Tables/SetupNewTab.h>
+#include <casacore/tables/Tables/TableRecord.h>
+#include <casacore/tables/Tables/TableDesc.h>
+#include <casacore/tables/Tables/ScaColDesc.h>
+#include <casacore/casa/Utilities/Assert.h>
+#include <casacore/casa/OS/Directory.h>
+#include <casacore/casa/iostream.h>
+
+#include <casacore/casa/namespace.h>
```

Comment:
[Google forbids importing an entire namespace.](https://google.github.io/styleguide/cppguide.html#Namespaces) Instead, only use what you need, for example:
```
using casacore::Table;
using casacore::TableDesc;
using casacore::TableUtil;
```

Reply:
Casacore forbids imports of class, function or namespace in header files, so anybody can safely uses them. But imports are allowed it in .cc files because that scope is limited.
I've changed the include into a normal using statement.

Reply:
For .cc files, Google also forbids importing an entire namespace. I don't know to what extend we want to follow Google style for Casacore, though.